docs/quick_start/clickHouse_file_example.md

```diff
@@ -0,0 +1,104 @@
+---
+title: File to ClickHouse 
+sidebar_position: 2
+---
+
+Here we use an example to introduce how to collect file and write to clickHouse.
```

Comment:
collect file and write to clickHouse.
->
create `File -> Pulsar -> ClickHouse` data ingestion.